File: code_combat/4_Backwoods_Forest/153-Reward_and_Ruination/reward.js
// It seems like the Ogre Chieftain is stealing your gems!
// Use the two artillery cannons for your goals.

while(true) {
    var enemy = hero.findNearestEnemy();
    if(enemy) {
        var enemyPos = enemy.pos.x + " " + enemy.pos.y;
        hero.say("Enemy at " + enemyPos);
    }
    // Now that you have sweet revenge,
    // why not have your cake and eat it, too?
    // Find the item's position and 
    // say it for your artillery to target.
    var item = hero.findNearestItem();
    if (item) {
        var itemPos = item.pos.x + " " + item.pos.y;
        hero.say("Item at " + itemPos);
    }
}

File: projects/code_combat/4_Backwoods_Forest/153-Reward_and_Ruination/reward.py
while True:
    enemy = hero.findNearestEnemy()
    if enemy:
        enemyPos = enemy.pos.x + " " + enemy.pos.y
        hero.say("Enemy at " + enemyPos)

    item = hero.findNearestItem()
    if item:
        itemPos = item.pos.x + " " + item.pos.y
        hero.say("Item at " + itemPos)
